### DISCUSSION TITLE
Error during converting data based on version 1.1

### DISCUSSION THREAD

**Discussion by JiSuanJiDaWang** [open-mmlab/mmdetection3d#2021] [Q&A] [answered]
I used the kitti converted using mmdetection3d-master on version 1.1 and found the error. So I used the create_data tools on version 1.1 to convert the data but found the following error:
![img_v2_a7fd74e4-b462-447d-9086-4cd8f298dd1g](https://user-images.githubusercontent.com/82489384/201264862-478247be-09fa-4314-a372-d18adc7c5135.png)

**Comment by Xiangxu-0103** [comment, marked as answer]
This has been fixed in #1923

**Reply by JiSuanJiDaWang** [reply]
Thanks
